composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require-dev": {
         "csa/guzzle-cache-middleware": "^1.0",
-        "elife/api": "^2.22",
+        "elife/api": "2.23.0",
```

Comment:
This should be `^2.23`

Can be achieved by `composer require elife/api`.

We may need to consider the impact of releasing the version bump to recommendations at a later date. Might make sense to work on recommendations schema and api-dummy changes first. Let's discuss later.